airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1683,13 +1683,17 @@ def create_empty_dataset(self, dataset_id="", project_id="",
                 'BigQuery job failed. Error was: {}'.format(err.content)
             )
 
-    def delete_dataset(self, project_id, dataset_id):
+    def delete_dataset(self, project_id, dataset_id, delete_contents):
```

Comment:
How about assigning a default `False` to delete_contents?

Reply:
Fixed and pushed.

Reply:
thanks @mohannadbanayosi